app/views/posts/_breadcrumbs.haml

```diff
@@ -1,3 +1,5 @@
+-# locals: ( post:, subtitle: '' )
```

Comment:
Any reason to prefer `''` and `.blank?` as the default instead of `nil` and checking `.nil?`?

Reply:
Because I was trying to match types at some point in the past xD